Converters/ArcGIS/Speckle.Converters.ArcGIS3/ToSpeckle/Raw/MultipatchFeatureToSpeckleConverter.cs

```diff
@@ -26,7 +26,8 @@ public IReadOnlyList<Base> Convert(ACG.Multipatch target)
   {
     List<Base> converted = new();
     // placeholder, needs to be declared in order to be used in the Ring patch type
-    SGIS.PolygonGeometry3d polygonGeom = new() { };
+    SGIS.PolygonGeometry3d polygonGeom =
+      new() { units = _contextStack.Current.Document.ActiveCRSoffsetRotation.SpeckleUnitString };
```

Comment:
Looks like it's being set once, calculated from `ACG.SpatialReference`, this `Document` is our own `ArcGISDocument` class (I presume because we need reference to both the Arcgis `Project`, `Map` and `CRSoffsetRotation`

This will need some cleanup with the `record` context changes that you and @adamhathcock have worked on. For now I'm tempted to leave as is.